AirCasting/SDSync/ViewsFlow/SDSyncUnplugAB/UnplugABView.swift

```diff
@@ -33,7 +33,7 @@ struct UnplugABView<VM: UnplugABViewModel>: View {
 extension UnplugABView {
     
     var unplugImage: some View {
-        Image("airbeam")
+        Image("airbeam-unplugged")
```

Comment:
It looks like this image is a bit smaller than the one on the screen before and on the screen after. If that could be changed, that would be nice, but if not, I think it's still okay 😊